includes/User.php

```diff
@@ -2551,6 +2551,13 @@ public function setGlobalPreference($preference, $value) {
 	}
 
 	/**
+	 * @param String $preference set to default
+	 * for now its support only old aproach to handle preferences (get/set Option)
+	 */
+	public function removeGlobalPreference($preference){
+		unset( $this->mOptions[ $preference ] );
```

Comment:
can we add a TODO here that says the user-preference service has a task for this: https://wikia-inc.atlassian.net/browse/SERVICES-483